DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: xportr
 Title: Utilities to Output CDISC SDTM/ADaM XPT Files
-Version: 0.3.0
+Version: 0.4.0
```

Comment:
Is this waiting for 0.4, or is it coming in with 0.3?

Reply:
current version is 0.2, so this should be 0.3, Thanks @EeethB